### PR TITLE
Merge Branch DAT-37 to Master

### PR DESCRIPTION
**Pull request by kramsey458**

### CODE REVIEW

File: python-flask/app.py [hunk 1]
```diff
@@ -56,7 +57,7 @@ def register():
             user['password'])
 
         user_exists = mongo.db.users.find_one({'email': user['email']}) != None
-        if(user_exists):
+        if (user_exists):
```
Comment (tonyc856) [inline python-flask/app.py:60]: 👍

File: python-flask/error.log [hunk 2]
```diff
@@ -0,0 +1 @@
+INFO:werkzeug: * Running on http://0.0.0.0:33507/ (Press CTRL+C to quit)
```
Comment (ZRich97) [inline python-flask/error.log:1]: Add to .gitignore

File: .travis.yml [hunk 3]
```diff
@@ -4,7 +4,8 @@ python:
     - "3.7-dev"
 
 install:
-    - pip install -r requirements.txt
+    - pip install pipenv
```
Comment (brobinson15) [inline .travis.yml:7]: pip pip

File: Pipfile [hunk 4]
```diff
@@ -43,6 +43,8 @@ PyJWT = "==1.7.0"
 Werkzeug = "==0.14.1"
 flask-cors = "*"
 gunicorn = "*"
+pandas = "*"
```
Comment (brobinson15) [inline Pipfile:46]: *trash-bears

File: python-flask/error.log (hunk 2 above)
Comment (brobinson15) [reply python-flask/error.log:1]: I concur doctor richardson

File: python-flask/app.py [hunk 5]
```diff
@@ -105,8 +105,29 @@ def unauthorized_response(callback):
     }), 401
 
 
+@app.route('/upload', methods=['POST'])
+def upload_file():
+    if request.method == 'POST':
+        files = request.files.to_dict()
+        for f in files:
```
Comment (tonyc856) [inline python-flask/app.py:112]: Just call it what it is, which is file in this case. It's just a personal style, not too big of a deal.

**Comment by kramsey458** [comment]
Outdated

**Pull request closed without merging**